# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-text/white-space/text-wrap-balance-line-clamp-001.html.ini
# Commit: 7ce2b41bb730
# Full Hash: 7ce2b41bb730247e00da9c69a1df9a01df80633d
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2023-09-28 21:51:27
# Regressor Bug: 1731541
# File Overlap Count: 1
# Description:
#   Bug 1731541 - When line-clamp is in effect, make text-wrap:balance consider only the lines up to the clamp limit. r=emilio
#   
#   This corresponds to how Chrome behaves, and passes the test they included in WPT.
#   
#   It's unclear to me whether this behavior actually follows from the current spec
# ==============================================================================

diff -r 71d889de8d97 -r 7ce2b41bb730 testing/web-platform/meta/css/css-text/white-space/text-wrap-balance-line-clamp-001.html.ini
--- a/testing/web-platform/meta/css/css-text/white-space/text-wrap-balance-line-clamp-001.html.ini	Thu Sep 28 08:31:21 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,2 +0,0 @@
-[text-wrap-balance-line-clamp-001.html]
-  expected: FAIL
